addons/gamification/models/res_users.py

```diff
@@ -353,6 +362,20 @@ def get_gamification_redirection_data(self):
         self.ensure_one()
         return []
 
+    def action_log_interacted_throttled(self):
+        """Log the user as active if wasn't already done in the last 5 minutes."""
+        now = datetime.datetime.now()
```

Comment:
`self.env.cr.now()` it better to have something consistent accross the current transaction